src/app/playlist/playlist.component.css

```diff
@@ -16,7 +16,7 @@
   flex-flow: column;
   text-align: left !important;
   width: 75%;
-  height: 80%;
+  height: 350px;
```

Comment:
Weird CSS bug on  the component force me to do it this way, ww can look deep into the reason, this is more like a "placeholder" height.